### ISSUE TITLE
Getting darker output images when fine-tuning the pre-trained ESRGAN on a custom dataset

### ISSUE THREAD

**Issue by Timmate** [idealo/image-super-resolution#176]
Hi @cfrancesco, I have tried using the ESRGAN pre-trained on DIV2K to fine-tune it on a custom dataset, but for some reason, the resulting SR images produced by the fine-tuned network visually seem to be of the same quality and detail as those produced by the pre-trained model, but in addition, they always appear slightly darker than the original HR images and images generated by the pre-trained ESRGAN. For my experiments, I have used the training settings provided in the ISR_Traininig_Tutorial.ipynb and tried to tweak them in various ways (e.g., raising flatness, decreasing the start learning rate to 0.001, employing more VGG layers in the feature extractor), but nothing has helped so far, I still keep getting darker images as an output. Do you have any idea what can cause this problem or what I am doing wrong in my experiments?

Best,
Tim

PS. Unfortunately, I am not allowed to share images from that custom dataset due to the NDA I have signed.

**Comment by gbriones1** [comment]
@Timmate  Can you share the code you are using?
